### DISCUSSION TITLE
Can Windows file explorer show the PDF files as the book cover? Instead of the Sumatra file icon.

### DISCUSSION THREAD

**Discussion by asafaq** [sumatrapdfreader/sumatrapdf#4731] [Q&A]
Since when I read them on my phone or tablet I do see the cover of every book, but in Windows 7 I can't.
Very nice program! Thank you devs!
I've done a mildly cover of the search results for similar discussions, it looks pretty hard to comprehend :/

**Comment by GitHubRulesOK** [comment]
Windows 7 is "technically unsupported by MS Windows!"
However is still the earliest supported by current versions including 3.6 pre-release (XP is now unsupported needs version 3.1.2)
Behaviour pre Windows 10 and soon pre 11 are tied to Windows Policy.

The cover images of a PDF or ePub can be shown if the pre-viewer is installed (NOT portable version)
![image](https://github.com/user-attachments/assets/7eeaaad7-790d-4570-ba84-4c35bac018de)

it also depends on folder settings so icons must be off and small icon overlay is optional too.
![image](https://github.com/user-attachments/assets/d6c7a025-c6c4-4d77-94bf-495f481533b7)
